### ISSUE TITLE
can i disable any axis?

### ISSUE THREAD

**Issue by iamisti** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#197]
is it possible to disable any axis with this plugin? I really only want rotation to work in my vr (playing in car racing games)

Thanks!

**Comment by Kung-vr** [comment]
If you mean cancel motion from any axis, unfortunately only yaw is feasible right now. Pitch and roll rotations of the universe center are ignored in favor of the calibrated imu down direction from the hmd (which is further enhanced with combination from controller imus)

The only way around this that I could see would be to write a custom driver to emulate the hmd's position with the added cancelled roll and pitch applied directly to the hmd rather than universe center. Openvr Input Emulator does some stuff like this, but because it isn't just using Openvr api calls it's more prone to breaking when steamvr changes. Right now I don't think there's anyone on the Advanced Settings team ready to take on the maintinance roll of something so prone to breaking like that.

**Comment by iamisti** [comment]
yeah thanks. I would only need pitch at least to be cancelled. Don't know how I'll do it then...

**Comment by iamisti** [comment]
closed?

**Comment by ykeara** [comment]
It is not feasible for us to do a motion cancellation feature w/ advanced settings.

It is outside the scope of the project (albeit I understand that is kinda nebulous at times.)

https://xsimulator.net/community/threads/open-vr-coder-needed-please-help.13226/ is a good forum thread discussing the issue.

Further you need the head movement still... it will be a terrible experience w/o it.

As kung said its a project better suited to https://github.com/matzman666/OpenVR-InputEmulator... I will be honest it interests me, but The algorithm needed to get a "good" result (w/ vibration) is fairly complex

tl;dr at this point in time its not feasible for advanced Settings, and is suited best for other projects.

**Comment by iamisti** [comment]
@ykeara thanks for the reply. I would really only want to disable some axis. 
At the end i don't want motion cancellation, i only want rotation to work (when i rotate my head left and right on the 'x' axis. All other movements can be set to 0.
Is this hard to do?
